uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/api/v3/LeaveRequestTest.php

```diff
@@ -1594,6 +1594,69 @@ public function testGetShouldOnlyReturnTheLeaveRequestsOfStaffMembersManagedByMa
     $this->assertEquals(0, $result['count']);
   }
 
+  /**
+   * @expectedException CiviCRM_API3_Exception
+   * @expectedExceptionMessage Mandatory key(s) missing from params array: leave_request_id
+   */
+  public function testIsManagedByShouldThrowAnExceptionIfLeaveRequestIDIsMissing() {
+    civicrm_api3('LeaveRequest', 'isManagedBy', ['contact_id' => 1]);
+  }
+
+  /**
+   * @expectedException CiviCRM_API3_Exception
+   * @expectedExceptionMessage Mandatory key(s) missing from params array: contact_id
+   */
+  public function testIsManagedByShouldThrowAnExceptionIfContactIDIsMissing() {
+    civicrm_api3('LeaveRequest', 'isManagedBy', ['leave_request_id' => 1]);
+  }
+
+  /**
+   * @expectedException CiviCRM_API3_Exception
+   * @expectedExceptionMessage Mandatory key(s) missing from params array: leave_request_id, contact_id
+   */
+  public function testIsManagedByShouldThrowAnExceptionIfBothTheContactIDAndLeaveRequestIDAreMissing() {
+    civicrm_api3('LeaveRequest', 'isManagedBy');
+  }
+
+  public function testIsManagedByShouldReturnTrueIfTheContactOfTheGivenLeaveRequestIsManagedByTheGivenContactID() {
```

Comment:
I'm seeing it from the perspective that the API returns false also in the same test function. 
It might be a way to be sure the method would not return true even if the relationship between the two contacts doesn't exist but since logically the API can return either true or false, its like both scenarios is being tested in this same function and the test description gives the impression that only the the scenario when the API is to return true is tested.